src/components/bulk-email-tool/bulk-email-task-manager/BulkEmailContentHistory.jsx

```diff
@@ -91,47 +90,38 @@ export function BulkEmailContentHistory({ intl }) {
         body={(
           <div>
             <div className="d-flex flex-row">
-              <p>
-                {intl.formatMessage(messages.modalMessageSubject)}
-              </p>
-              <p className="pl-2">
-                {messageContent.subject}
-              </p>
+              <p>{intl.formatMessage(messages.modalMessageSubject)}</p>
+              <p className="pl-2">{messageContent.subject}</p>
             </div>
             <div className="d-flex flex-row">
-              <p>
-                {intl.formatMessage(messages.modalMessageSentBy)}
-              </p>
-              <p className="pl-2">
-                {messageContent.requester}
-              </p>
+              <p>{intl.formatMessage(messages.modalMessageSentBy)}</p>
+              <p className="pl-2">{messageContent.requester}</p>
             </div>
             <div className="d-flex flex-row">
-              <p>
-                {intl.formatMessage(messages.modalMessageTimeSent)}
-              </p>
-              <p className="pl-2">
-                {messageContent.created}
-              </p>
+              <p>{intl.formatMessage(messages.modalMessageTimeSent)}</p>
+              <p className="pl-2">{messageContent.created}</p>
             </div>
             <div className="d-flex flex-row">
-              <p>
-                {intl.formatMessage(messages.modalMessageSentTo)}
-              </p>
-              <p className="pl-2">
-                {messageContent.sent_to}
-              </p>
+              <p>{intl.formatMessage(messages.modalMessageSentTo)}</p>
+              <p className="pl-2">{messageContent.sent_to}</p>
             </div>
             <hr className="py-2" />
             <div>
-              <p>
-                {intl.formatMessage(messages.modalMessageBody)}
-              </p>
+              <p>{intl.formatMessage(messages.modalMessageBody)}</p>
+              {/* eslint-disable-next-line react/no-danger */}
               <div dangerouslySetInnerHTML={{ __html: messageContent.email.html_message }} />
             </div>
           </div>
         )}
         onClose={() => setIsMessageModalOpen(false)}
+        buttons={[
+          <Button onClick={() => {
+            copyTextToEditor(messageContent.email.subject, messageContent.email.html_message);
```

Comment:
Based on our conversation during standup yesterday, should the email's subject still be passed to this function? For some reason I thought we were only going to use the message content?